services/amundi-auth/main.py

```
@@ -0,0 +1,683 @@
+    for state in states:
+        await _dispose_pending_state(state)
+
+
+async def _first_visible(page: Page, selectors: list[str]):
+    for selector in selectors:
+        locator = page.locator(selector).first
+        try:
+            if await locator.is_visible(timeout=800):
+                return locator
+        except Exception:
+            continue
```

Ruff (0.16.1)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)